app/.classpath

```diff
@@ -6,6 +6,10 @@
 	<classpathentry exported="true" kind="con" path="com.android.ide.eclipse.adt.DEPENDENCIES"/>
 	<classpathentry exported="true" kind="con" path="com.android.ide.eclipse.adt.ANDROID_FRAMEWORK"/>
 	<classpathentry combineaccessrules="false" exported="true" kind="src" path="/javarosa-core"/>
+	<classpathentry exported="true" kind="lib" path="libs/commons-codec.jar"/>

```

Comment:
Now that we've moved to using /lib/ I don't think we need to manually include these in the classpath anymore.